x-pack/filebeat/input/http_endpoint/input.go

```diff
@@ -391,17 +402,21 @@ func newInputMetrics(id string) *inputMetrics {
 		apiErrors:           monitoring.NewUint(reg, "api_errors_total"),
 		batchesReceived:     monitoring.NewUint(reg, "batches_received_total"),
 		batchesPublished:    monitoring.NewUint(reg, "batches_published_total"),
+		batchesACKedTotal:   monitoring.NewUint(reg, "batches_acked_total"),
```

Comment:
Reminder to add the new metrics to the table in the docs.

Reply:
I just noticed that the lumberjack input does not have any docs.

Reply:
See my comment under _Author's Notes_ at https://github.com/elastic/beats/pull/32175#issue-1290670673. Whether or not to formally advertise it is something I still debate. Feedback welcomed. It is only used and exposed as part of the barracuda integration at the moment.

Reply:
Thanks, that makes perfect sense.